socorro/external/es/super_search_fields.py

```diff
@@ -1448,6 +1448,34 @@ def number_field(
         "query_type": "enum",
         "storage_mapping": {"type": "string"},
     },
+    "crashing_thread_id": {
```

Comment:
Use `number_field` because it sets up the defaults. Here's an example:

https://github.com/mozilla-services/socorro/blob/6e7aadc03a2ba5315a978fac433285dde8758af3/socorro/external/es/super_search_fields.py#L1895-L1903